store/store_test.go

```diff
@@ -3289,19 +3652,43 @@ func (s *storeTestSuite) TestFind500once(c *C) {
 	cfg := store.Config{
 		StoreBaseURL: mockServerURL,
 		DetailFields: []string{},
+		SearchFields: []string{},
 	}
 	sto := store.New(&cfg, nil)
 
 	snaps, err := sto.Find(s.ctx, &store.Search{Query: "hello"}, nil)
 	c.Check(err, IsNil)
 	c.Assert(snaps, HasLen, 1)
 	c.Assert(n, Equals, 2)
+	if apiV1 {
+		c.Check(v1Fallback, Equals, true)
+	} else {
+		c.Check(v2Hit, Equals, true)
+	}
+}
+
+func (s *storeTestSuite) TestFindV1_500once(c *C) {
```

Comment:
if not  s/once/Once/

Reply:
Right, sorry I missed them before, I think you already pointed those out in the 1st pass.